pyre/extra_tests/pip/run.py

```
@@ -0,0 +1,704 @@
+
+
+def check_uninstall(ctx: Context) -> None:
+    """Uninstalling removes the modules and the scripts that came with them."""
+    result = _ok(
+        _spawn(ctx, [ctx.python, "-P", "-m", "pip", "uninstall", "-y", "stpkg", "tinypkg"], cwd=ctx.root)
+    )
+    _contains(result, "Successfully uninstalled stpkg-0.2.0")
+    _contains(result, "Successfully uninstalled tinypkg-0.1.0")
+    gone = _spawn(ctx, [ctx.python, "-P", "-c", "import stpkg"], cwd=ctx.root)
+    if gone.returncode == 0:
+        raise Failed("the module still imports after being uninstalled", gone.describe())
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)